Example-Audit-REXX/README.md

```diff
@@ -0,0 +1,125 @@
+## Example-Audit-REXX
+
+This samples goes to a CPC and lists the attributes, number of general-purpose CPs, zIIPs, storage etc...,for all **operating** LPARs. It then stores the results, which are in .csv format, in a member in a z/OS data set.  You can then import the content into Microsoft™ Excel or another application of your choice.
```

Comment:
Instead of "... in a member in a z/OS data set. You can ...." maybe " ... in a z/OS data set member. You can ..."

Reply:
good call! done